README.md

```diff
@@ -1,8 +1,63 @@
+<<<<<<< HEAD
 # JanusGraph Docker images
 
+Repository for building and publishing [JanusGraph][JG] docker images.
+
 ## License
 
 JanusGraph Docker images are provided under the [Apache 2.0
 license](APACHE-2.0.txt) and documentation is provided under the [CC-BY-4.0
 license](CC-BY-4.0.txt). For details about this dual-license structure, please
 see [`LICENSE.txt`](LICENSE.txt).
+
+## Building ##
```

Comment:
How about moving the `Building` and `Publishing` section into a different document as it's only interesting to contributors but not normal users?